### ISSUE TITLE
denoising train

### ISSUE THREAD

**Issue by dorbarber79** [qibao77/CFSNet#6]
hello and thank you for your nice work.
during training of gray denoise the model is not converge if i use α_in= 0. only if i use α_in= 1 the loss converges and reduce
I tied several datasets and also with per-trained model and without.

thanks in advance for your help

**Comment by dorbarber79** [comment]
in your denoise instructions you write:
step 1 
noise_level 25 
no pre-trained model
input_alpha 0 

step  2
noise_level 50
use pre-trained model
input_alpha 1

my problem is that in step 1 (input_alpha 0 ) the model loss is not reducing

**Comment by qibao77** [comment]
I‘m sorry for late reply, has your problem been solved? I ran the code again in the denoising task, no problem. Can you provide more details?
